src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java

```diff
@@ -69,15 +122,55 @@ public void testImportEntries() throws IOException {
+
         BibtexEntry one = entries.get(0);
 
         Assert.assertEquals("Computing and operational research at the London Hospital", one.getField("title"));
+        Assert.assertEquals("Barber, Barry and Abbott, W.", one.getField("author"));
+        Assert.assertEquals("Computers in medicine series", one.getField("series"));
+        Assert.assertEquals("London :  Butterworths", one.getField("publisher"));
+        Assert.assertEquals("x, 102p, leaf : ill., form, port ; 22cm", one.getField("physicaldimensions"));
+        Assert.assertEquals("0407517006 (Pbk)", one.getField("isbn"));
+        Assert.assertEquals("Bibl.p.94-97. - Index", one.getField("note"));
+        Assert.assertEquals(
+                "London Hospital and Medical College, Electronic data processing - Medicine, Computers - Hospital administration, Hospital planning, Operations research, Hospital equipment and supplies, Electronic data processing - Hospitals - Administration, Hospitals, London, London Hospital and Medical College, Records management, Applications of computer systems, to 1971",
+                one.getField("keywords"));
+        Assert.assertEquals("TXT, PDF", one.getField("documenttype"));
+        Assert.assertEquals(
+                "Aberdeen ; Birmingham ; Edinburgh ; Trinity College Dublin ; UCL (University College London)",
+                one.getField("HL"));
+
+        Assert.assertEquals(BibtexEntryTypes.BOOK, one.getType());
 
         BibtexEntry two = entries.get(1);
 
         Assert.assertEquals("Real time systems : management and design", two.getField("title"));
+        Assert.assertEquals("Tebbs, David and Collins, Garfield", two.getField("author"));
+        Assert.assertEquals("London ; New York :  McGraw-Hill", two.getField("publisher"));
+        Assert.assertEquals("1977", two.getField("year"));
+        Assert.assertEquals("ix, 357p : ill., forms ; 24cm", two.getField("physicaldimensions"));
+        Assert.assertEquals("0070844828", two.getField("isbn"));
+        Assert.assertEquals("Index", two.getField("note"));
+        Assert.assertEquals("Real-time data processing - Management, Real time computer systems, Design",
+                two.getField("keywords"));
+        Assert.assertEquals("TXT, PDF", two.getField("documenttype"));
+        Assert.assertEquals(
+                "Aberdeen ; Birmingham ; Edinburgh ; Imperial College ; Liverpool ; Manchester ; Oxford ; Trinity College Dublin",
+                two.getField("HL"));
+
+        Assert.assertEquals(BibtexEntryTypes.BOOK, two.getType());
+    }
+
+    @Test
+    public void testImportEntries3() throws IOException {
+        CopacImporter importer = new CopacImporter();
+
+        //CopacImporterTest3.txt is an empty file.
+        List<BibtexEntry> entries = importer.importEntries(
+                CopacImporterTest.class.getResourceAsStream("CopacImporterTest3.txt"), new OutputPrinterToNull());
+        Assert.assertEquals(0, entries.size());
```

Comment:
When comparing lists, it is better to do an assert that way:

`Assert.assertEquals(Collections.emptyList(), entries);`

Then, the message when the assertion fails is much better, as it contains the entries in the list using their toString method. 

Example:

```java
List<String> values = new LinkedList<>();
values.add("asdf");

Assert.assertEquals(Collections.emptyList(), values);
```

results in 

```
Exception in thread "main" java.lang.AssertionError: expected:<[]> but was:<[asdf]>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at Test.main(Test.java:13)
```